external/storm-cassandra/src/main/java/org/apache/storm/cassandra/bolt/BaseCassandraBolt.java

```diff
@@ -105,17 +109,43 @@ public BaseCassandraBolt withResultHandler(ExecutionResultHandler resultHandler)
         return this;
     }
 
+    /**
+     * Configures this bolt with the given {@code fields} as outputfields with stream id as {@link Utils#DEFAULT_STREAM_ID}
+     *
+     * @param fields outputfields
+     */
     public BaseCassandraBolt withOutputFields(Fields fields) {
+        Preconditions.checkNotNull("fields should not be null.");
```

Comment:
Missing args, to -> `Preconditions.checkNotNull(fields, "fields should not be null.");`